snap/squashfs/stat_test.go

```diff
@@ -0,0 +1,64 @@
+package squashfs
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	. "gopkg.in/check.v1"
+)
+
+func (s *SquashfsTestSuite) TestStatModeBits(c *C) {
```

Comment:
I wonder if we should test this a bit more. Some of the error conditions, maybe with more than a single mocked output line?